test/common/gitops_utils.go

```diff
@@ -20,18 +20,16 @@ import (
 
 // GitOpsUserSetup configures a new user to use for the GitOps tests.
 // It returns the OCPUser instance, which contains a path to the created kubeconfig file.
-func GitOpsUserSetup() OCPUser {
+func GitOpsUserSetup(ocpUser *OCPUser) {
```

Comment:
Since the instance isn't returned, this comment should be updated.

Reply:
Good point! Done!